_data/projects/Maven-Using-CMD.yml

```diff
@@ -7,6 +7,6 @@ tags:
 - XML
 - Jenkins
 - Automation
-upforgrabs:
+Up-for-Grabs:
```

Comment:
@NishkarshRaj Please change this back, this is not user-defined. It is predefined.